UKHomeOffice/sgmr-service
# SPCR-395 Add customs questions to the voyage journey

### AC / Description of changes


### To Test
- create a voyage
- when asked if employed people say 'no'
- click save and continue
- > you should see the customs page
- go back
- > you should be returned to is anyone employed page
- select 'yes'
- select employed people
- click save and continue
- > you should see the customs page
- click save and continue
- > all fields should error
- refresh page
- click save and exit
- > all fields should error
- select 'yes' to all the question
- click save and continue
- > the two questions with textareas should error
- type in those fields
- refresh the page
- > answers should be persisted
- change one of those fields to 'no'
- change it back to 'yes'
- > textarea should be null, it should not persist what you typed when you've changed the answer to no
- click submit
- > Voyage form should be PATCHED with your answers
- > You should be taken to the Skipper details page
- go back
- > your answers should be persisted
- compete the form up to the check your answers page
- > you should see a section for Customs questions
- go back to the customs form
- > **paste** a lot of text into one of the textareas
- save and continue
- > you should return to the check your answers page
- > your pasted text should be visible
- submit the form
- > the form should submit

Links
- should all open in a new tab


### Notes

---

**Remember to merge to feature branches not `master`.**
